src/main/java/seedu/storage/Storage.java

```diff
@@ -2,30 +2,36 @@
 
 import com.google.gson.Gson;
 import com.google.gson.reflect.TypeToken;
-import seedu.equipment.DuplicateSerialNumber;
+import seedu.equipment.DuplicateSerialNumberException;
 import seedu.equipment.Equipment;
 import seedu.equipment.EquipmentManager;
-
-import java.io.*;
+import java.io.IOException;
+import java.io.Reader;
+import java.io.Writer;
 import java.nio.file.Files;
 import java.nio.file.Path;
 import java.util.ArrayList;
 import java.util.List;
 
 
 public class Storage {
+    public final String FILE_NOT_FOUND_ERROR_MESSAGE = "File not found! A new file will be created after this session.";
+    public final String SAVE_ERROR_MESSAGE = "An error occurred while saving!";
+    public final String DUPLICATE_SERIAL_NUMBER_ERROR = "Duplicate serial number found!";
     public final String path = "./equipments.json";
     private final Gson gson = new Gson();
 
-    public void loadData(EquipmentManager equipmentManager) throws DuplicateSerialNumber {
+    public void loadData(EquipmentManager equipmentManager) {
         try {
             Reader reader = Files.newBufferedReader(Path.of(path));
             List<Equipment> equipments = gson.fromJson(reader, new TypeToken<List<Equipment>>() {}.getType());
             for (Equipment equipment : equipments) {
                 equipmentManager.addEquipment(equipment);
             }
         } catch (IOException e) {
-            System.out.println("io1");
+            System.out.println(FILE_NOT_FOUND_ERROR_MESSAGE);
+        } catch (DuplicateSerialNumberException e) {
+            System.out.println(DUPLICATE_SERIAL_NUMBER_ERROR);
```

Comment:
What happens if there is indeed a `DuplicateSerialNumberException`? Are the earlier equipment added? If so, how do we want to handle this case? @Bryan-BC @yuhaochua 

I'm thinking we can just reset equipmentManager or perhaps print a message telling the user this?